pages/register/success.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react'
+import { useRouter } from 'next/router'
+import { motion } from 'framer-motion'
+import Head from 'next/head'
+import Image from 'next/image'
+
+const Success = () => {
+  const router = useRouter()
+  const { title } = router.query
+  useEffect(() => {
+    setTimeout(() => {
+      router.push('/events')
```

Comment:
dont' push. leave it on this page.

Reply:
Utkarsh bhaiya only said that, wait for 3 seconds and go to events